packages/broker/src/config/ConfigWizard.ts

```diff
@@ -1,254 +1,544 @@
+        )
+
+        log(`
+            >
+            > ~ *Congratulations, you've setup your Streamr node!*
+            > Your node address is ${chalk.greenBright(nodeAddress)}
+            > Your node's generated name is ${chalk.greenBright(getNodeMnemonic(privateKey))}
+        `)
+
+        if (operator) {
+            const resume = progress((f) =>
+                style(`> Your node address has *${f} MATIC* _– checking balance…_`)
+            )
+
+            try {
+                const balance = await getNativeBalance(network, nodeAddress)
+
+                const content = `Your node address has *${utils
+                    .formatEther(balance)
+                    .replace(/\.(\d+)/, (f) => f.substring(0, 3))} MATIC*`
+
+                resume()
+
+                if (balance.lt(MinBalance)) {
+                    log(`
+                        > ! ${content}. You'll need to fund it with a small amount of MATIC tokens.
+                    `)
+                } else {
+                    log(`> ${content}`)
+                }
+            } catch (e) {
+                resume()
+
+                log('> x Failed to fetch node\'s balance')
+            }
+        }
+
+        if (operator) {
+            log('> ')
+
+            const resume = progress((f) =>
+                style(`
+                    > _Checking if your node has been paired with your Operator… ${f}_
+                `)
+            )
+
+            try {
+                const nodes = await getOperatorNodeAddresses(network, operator)
+
+                resume()
+
+                const hub =
+                    network === 'polygon'
+                        ? 'https://streamr.network/hub'
+                        : 'https://mumbai.streamr.network/hub'
 
-export interface PluginAnswers extends Answers {
-    enabledApiPlugins: string[]
-    websocketPort?: string
-    mqttPort?: string
-    httpPort?: string
+                if (nodes) {
+                    if (!nodes.includes(nodeAddress.toLowerCase())) {
+                        log(
+                            '> ! You will need to pair your node with your Operator:'
+                        )
+                    } else {
+                        log('> Your node has been paired with your Operator:')
+                    }
+                } else {
+                    log(`
+                        > x Your Operator could not be found on the **${network.replace(/\w/, (l) => l.toUpperCase())}** network, see
+                    `)
+                }
+
+                log(`> *${hub}/network/operators/${operator}*`)
+            } catch (e) {
+                resume()
+
+                log('> x Failed to fetch operator nodes')
+            }
+        }
+
+        log(`
+            >
+            > You can start your Streamr node now with
+            > *streamr-broker ${storagePath}*
+            >
+            > For environment specific run instructions, see
```

Comment:
Which error case this handles ? E.g. ctrl+c or something else?

Reply:
Ctrl+c, yes.

Reply:
Does the throw error /force closed/ come from the inquired library? Could add a comment there

(Refering to line 172 here, line numbers corrupted again, maybe because of the force push or something)